using_images/s2i_images/php.adoc

```diff
@@ -124,6 +124,13 @@ The following environment variable sets its equivalent property value in the opc
 |16M
 |===
 
+ifdef::openshift-origin[]
+|`*OPCACHE_REVALIDATE_FREQ*`
+|How often to check script timestamps for updates, in seconds. 0 will result in OPcache checking for updates on every request.
```

Comment:
We dont explain the environame variable(except the APP_CONFIG in python, but thats a bit trickier) Does it make sense to link it ? If the user searches for it (or any other env var we expose) first hit is the right one.